docs/building/windows-core.md

```diff
@@ -44,10 +44,7 @@ Invoke-WebRequest -Uri https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/sc
 If you have any problems installing `dotnet`,
 please see their [documentation][cli-docs].
 
-If you are using Windows 7, Windows Server 2008 or Windows Server 2012
-you will also need to install
-[Visual C++ Redistributable for Visual Studio 2012 Update 4][redist-2012]
-and [Visual C++ Redistributable for Visual Studio 2015][redist-2015].
+You will also need to install a recent version of Visual Studio with the optional feature 'Common Tools for Visual C++' installed. The free [Community edition of Visual Studio 2015](https://beta.visualstudio.com/visual-studio-community-vs/) should work fine.
```

Comment:
The requirements match those of [CoreCLR](https://github.com/dotnet/coreclr/blob/master/Documentation/building/windows-instructions.md), so we should reflect that here:
1. Visual Studio 2015 (Community, Enterprise, Professional)
2. "Common Tools for Visual C++ 2015"

Side Note:
If you have VS 2015 installed, you should not need the redistributable package to build, so it should be safe to remove that dependency from the documentation. However, there is a runtime dependency for pwrshplugin.dll on downlevel systems. It can either be satisfied by installing the VC++ redistributable package or fully patching the machine to pull in the appropriate KB. I'm not sure if that is documented in our MDs.